protocol_grpc.go

```diff
@@ -843,28 +843,35 @@ func grpcErrorToTrailer(trailer http.Header, protobuf Codec, err error) {
 	}
 	status := grpcStatusFromError(err)
 	code := strconv.Itoa(int(status.GetCode()))
-	bin, binErr := protobuf.Marshal(status)
-	if binErr != nil {
-		setHeaderCanonical(
-			trailer,
-			grpcHeaderStatus,
-			strconv.FormatInt(int64(CodeInternal), 10 /* base */),
-		)
-		setHeaderCanonical(
-			trailer,
-			grpcHeaderMessage,
-			grpcPercentEncode(
-				fmt.Sprintf("marshal protobuf status: %v", binErr),
-			),
-		)
-		return
+	var bin []byte
```

Comment:
Above we always set the Grpc-Message header on nil errors but looks like this is optional too? Safe to remove @jhump ?